docs/refs.bib

```diff
@@ -38,3 +38,53 @@ @inproceedings{hardt2016equality
   year      = {2016},
   url       = {https://proceedings.neurips.cc/paper/2016/hash/9d2682367c3935defcb1f9e247a97c0d-Abstract.html}
 }
+
+@inproceedings{10.1145/3287560.3287598,
+author = {Selbst, Andrew D. and Boyd, Danah and Friedler, Sorelle A. and Venkatasubramanian, Suresh and Vertesi, Janet},
+title = {Fairness and Abstraction in Sociotechnical Systems},
+year = {2019},
+isbn = {9781450361255},
+publisher = {Association for Computing Machinery},
+address = {New York, NY, USA},
+url = {https://doi.org/10.1145/3287560.3287598},
+doi = {10.1145/3287560.3287598},
+abstract = {A key goal of the fair-ML community is to develop machine-learning based systems that, once introduced into a social context, can achieve social and legal outcomes such as fairness, justice, and due process. Bedrock concepts in computer science---such as abstraction and modular design---are used to define notions of fairness and discrimination, to produce fairness-aware learning algorithms, and to intervene at different stages of a decision-making pipeline to produce "fair" outcomes. In this paper, however, we contend that these concepts render technical interventions ineffective, inaccurate, and sometimes dangerously misguided when they enter the societal context that surrounds decision-making systems. We outline this mismatch with five "traps" that fair-ML work can fall into even as it attempts to be more context-aware in comparison to traditional data science. We draw on studies of sociotechnical systems in Science and Technology Studies to explain why such traps occur and how to avoid them. Finally, we suggest ways in which technical designers can mitigate the traps through a refocusing of design in terms of process rather than solutions, and by drawing abstraction boundaries to include social actors rather than purely technical ones.},
```

Comment:
But... shouldn't we? Do we really want to keep abstracts of papers here? It's not like we're going to render those...

Reply:
We could trim it; I was just doing the simplest thing based on the citation source.

Reply:
If we're not rendering it let's just cut it(?)

Reply:
I'm happy to have as few modifications as possible on the copy pasted chunk. Also, this doesn't take that much space, why bother.

Reply:
Since we are providing URL, I would personally lean towards skipping most of the extra meta-information here like isbn, abstract, numpages, keywords. But I don't care too much about extra meta-information that doesn't get displayed.

Two things that I think we should care about are:
* consistency in the naming of the bibtex ids (format <author><year><keyword>, where keyword is most of the time the first word in the paper title)
* consistency in the appearance of the references at the end of files (this is something that is actually not too hard to fix later, because we only need to edit .bib file...)
* linking to peer-reviewed versions of papers rather than arXiv whenever peer-reviewed versions are available

Reply:
Skipping metainformation is the business of the bibstyle file. Isn't that pretty much the whole point of BibTeX?

Reply:
The only issue that I see re. consistency is that these bibtex entries are generating multiple URLs when rendered. I'm in favor of dropping "eprint" and "doi" and retaining URL. (I think that's simpler that trying to tweak the stylefile?)